content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -2,94 +2,98 @@
-all of the components and for each component, indicate which variant it applies to, 
-using the *Apply on Variants* column, as shown below. 
+You can create one BoM per variant by specifying the *Product Variant* in the dedicated field below
+the product name. Alternatively, you can use one master BoM that contains all of the components, and
+specify which variant each component applies to using the *Apply on Variants* column, as shown
+below. 
 
 .. image:: bill_configuration/bom_2.png
     :align: center
+    :alt: Product Variants in the Bill of Materials. 
 
 
 Adding Operations
 =================
 
-You can also add operations to your *BoM*, if you want workers to follow instructions
-or register time spent. To use this feature, enable the *Work Orders* feature in the 
-*Manufacturing* app settings, as shown below.
+You can also add *Operations* to your BoM if you want workers to follow instructions or register
+time spent. To use this feature, enable the *Work Orders* feature in the *Manufacturing* app
+settings, as shown below.
 
 .. image:: bill_configuration/bom_3.png
     :align: center
+    :alt: Work Orders Setting. 
 
 .. note::
-         Each operation is unique as it is always linked to only one BOM. This being said, 
-         Operations Operations can be re-used when configuring a new BOM, with the *Copy Existing 
+         Each operation is unique, as it is always exclusively linked to one BoM. 
+         Operations can be re-used when configuring a new BoM, with the *Copy Existing 
          Operations* feature.
 
 .. image:: bill_configuration/bom_4.png
     :align: center
+    :alt: *Copy Existing Operations* feature. 
 
-Finally, just like for components, operations can also be variant specific only, as shown below.
+Like components, operations can also be variant-specific, as shown below.
 
 .. image:: bill_configuration/bom_5.png
     :align: center
+    :alt: Variant-specific operations. 
 
 
 
 Adding By-Products
 ==================
 
-A *by-product* is a product that is produced on top of the main product 
-of a *BoM*. As opposed to the primary product, there can be more than one on a BOM. 
+A *By-Product* is a product that is produced in addition to the main product of a BoM. Unlike the
+primary product, there can be more than one by-product on a BoM. 
 
-To add *by-products* to a *BoM*, you will first need to enable the by-product
-feature from the *Manufacturing* app settings.
+To add by-products to a BoM, you will first need to enable the *By-Products* feature from the
+*Manufacturing* app settings.
 
 .. image:: bill_configuration/bom_6.png
     :align: center
+    :alt: By-Products setting. 
 
-Once the feature is enabled, you can add *by-products* to your
-*BoMs*. Note that if you have Operations, you'll need to specify 
-in which operation the by-product is produced. 
+Once the feature is enabled, you can add by-products to your BoMs. If you have operations, you'll
```

Comment:
- Delete `you can` 
- Replace `your BoMs. If you have operations, you'll need to specify exactly which operation the by-product will be produced from.` with `a BoM by clicking on the :guilabel:'Operations' tab and clicking :guilabel:'Add a line'. Then, name the by-product, and specify the :guilabel:'Quantity' and the :guilabel:'Unit of Measure'. If the BoM has configured operations, specify exactly which operation the by-product is produced from in the :guilabel:'Produced in Operation' field. Finally, click :guilabel:'Save'.`

Note: in my suggested changes, single quotation marks ( ' ) = backticks ( ` ), I use single quotation marks in review comments so it will actually show on Github